keyboards/handwired/hnah108/hnah108.h

```diff
@@ -40,14 +40,14 @@
 }
 
 #define LAYOUT_fullsize_iso( \
-    k00,      k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k60, k61,     k62, k63, k64,                          \
+    k00,      k01, k02, k03, k04, k05, k06, k07, k08, k09, k0A, k60, k61,     k62, k63, k64,      k65, k66, k67, k68, \
```

Comment:
This change is incompatible with the `fullsize_iso` community layout declared in the keyboards `rules.mk`

Potential solutions:
1. keep the existing macros as is and add new macros added which enable the use of the extra 4 keys
1. remove community support

Reply:
thank you